CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [Unreleased]
 
+### Added
+
+- `regime`: added a template regime.
+
+### Changed
+
+- `tools`: add explicit dependency for golangci-lint
+- `docs`: add `CONTRIBUTING.md` file
```

Comment:
These are additions as opposed to changes. We use the backticks for package names, I don't think that's needed here.